### DISCUSSION TITLE
Menu entry towards config directory

### DISCUSSION THREAD

**Discussion by BNolet** [espanso/espanso#1341] [Feature requests and Ideas]
Hi there, I found [this discussion](https://github.com/espanso/espanso/discussions/1234), so I understand we'll be able to edit the configs through a GUI at some point. I would like to request as a stopgap measure that a menu entry that opens the config file directory be added.

I figure it would be a relatively trivial change to tide folks over until the GUI is put in

**Comment by Chaoscontrol** [comment]
This tbh is my only bother for Espanso. I don't modify or add new matches just for the hassle of going and finding the damn file hidden in the file structure. 
Just a menu item to open the config folder would be great.

**Comment by BNolet** [comment]
Might see if I can do this myself since it's "relatively trivial" as per my words :P 

I've only got *some* experience in Rust so here's hoping I'm not diving into a rabbit hole

**Reply by Chaoscontrol** [reply]
It can't be they hard... Can it? It's just one extra button in an already existing menu, with a link into a static path for each of the supported OS.

I clueless about coding though. Hah.

**Comment by BNolet** [comment]
I've gotten it working in this PR: https://github.com/espanso/espanso/pull/1870

Looking forward to seeing whether this actually works on Windows and Linux too. I couldn't manage to get it to build on Windows 11 unfortunately, so I'll have to wait until someone else can to test

**Reply by Chaoscontrol** [reply]
This is amazing! Thank you so much!
I use Windows 11 but I am clueless as to how to build and test this.

**Comment by BNolet** [comment]
This will be added in the next official release! :D
